src/plugins/fs/generic.c

```diff
@@ -1520,3 +1598,46 @@ gboolean bd_fs_mkfs (const gchar *device, const gchar *fstype, BDFSMkfsOptions *
 
     return ret;
 }
+
+
+/**
+ * bd_fs_features:
+ * @fstype: name of the filesystem to get features for (e.g. "ext4")
+ * @error (allow-none): (out): place to store error (if any)
+ *
+ * Returns (transfer-none): features supported by @fstype, see %BDFSFeatures for more information.
+ *
+ * Tech category: always available
+ *
+ */
+const BDFSFeatures* bd_fs_features (const gchar *fstype, GError **error) {
+    if (g_strcmp0 (fstype, "exfat") == 0) {
+        return &fs_features[BD_FS_TECH_EXFAT - BD_FS_OFFSET];
```

Comment:
I think it would be nicer to have a function that would return `BD_FS_TECH_EXFAT` for `"exfat"` and use it here and possibly in other places.

Reply:
macros?

Reply:
I still think that function would make this nicer.

Reply:
Yes, I ignored this one for now, I wanted to fix the "obvious" errors first. Are you thinking about a public function/macro or just something for internal use?